rts/motoko-rts/src/float.rs

```diff
@@ -6,33 +6,34 @@ use motoko_rts_macros::ic_mem_fn;
 
 // The meaning of the `mode` parameter is documented in motoko-base, function Float.format()
 #[ic_mem_fn]
-unsafe fn float_fmt<M: Memory>(mem: &mut M, a: f64, prec: u32, mode: u32) -> Value {
-    // prec and mode are tagged small words (`Nat8`s), so we shift 24 bits. See
+unsafe fn float_fmt<M: Memory>(mem: &mut M, a: f64, prec: usize, mode: usize) -> Value {
```

Comment:
Probably not worth fixing, but maybe we should do the decoding of the scalars on the Motoko size, before calling float_fmt, to isolate the rts from the scalar representation choice.

Reply:
Indeed, this would be better, also for maintainability.